.circleci/config.yml

```diff
@@ -117,25 +98,18 @@ workflows:
           requires:
             - cross_build
       - perf_test:
-          name: perf-test-flux
-          format: flux-http
-          record_ingest_results: true
+          #FIXME: replaced config to do dry run on pr
+          record_results: false
           requires:
             - cross_build
-          filters:
-            branches:
-              only:
-                - "master"
-      - perf_test:
-          name: perf-test-influxql
-          format: http
```

Comment:
flux and influxql are now tested during the same script run.

Reply:
We had split these up in an effort to parallelize and run things faster...but like my comment above alludes to; it shouldn't really matter since they only run on master.

If we ever want to run these on PRs and possibly gate PRs based on the results, we'll need to figure out a way to make them go faster, but that's a totally different discussion.

Reply:
I think ideally we would be able to spin up separate AWS instances for each category of tests or something similar - ideally the split would be based on dataset rather than by [flux vs influxql]. Before I merged these back together we had to pay the cost of installing each dataset twice.

For now I value simplicity, but in the future yeah it would be great to split up and run multiple tests in parallel. I think having this configuration driven will make that easier.